app/packs/src/apps/mydb/elements/details/reactions/analysesTab/ReactionDetailsContainers.js

```diff
@@ -256,8 +257,11 @@ export default class ReactionDetailsContainers extends Component {
       }
 
       return (
-        <div className="analysis-header order" style={{ width: '100%' }}>
-          <div className="preview">
+        <div
+          className="d-flex w-100 mb-0"
+          style={{ backgroundColor: '#ececec' }}
```

Comment:
We need to introduce proper helper classes for grey backgrounds – we use them all over the place...